js/views-manager.js

```diff
@@ -148,7 +148,8 @@ var ViewsManager = {
       });
       sortedFavedProjects.reverse(); // start from most recently favourited project
       // create a new div that shows count of favs
-      $("<div id='fav-count'><img src='svg/icon-fav.svg' width=20 />You have "+_.size(sortedFavedProjects)+" favs</div>").insertBefore(".projects");
+      var favsCount = _.size(sortedFavedProjects);
+      $("<div id='fav-count-text'><img class='star'></img>You have <span id='fav-count'>"+favsCount+"</span> favs</div>").insertBefore(".projects");
```

Comment:
2 things for the use of `<img class='star'></img>`

1. `<img>` is a [void element](http://w3c.github.io/html/syntax.html#void-elements) - it doesn't require end tag. 
2. `<img>` should have its `src` attribute set.

Let's stay with the original code `<img src='svg/icon-fav.svg' width=20>` 😉 

let's use `<img src='svg/icon-fav.svg' width=20 />`